tox.ini

```diff
@@ -1,5 +1,6 @@
 [tox]
 envlist=
+    py37-core
```

Comment:
To provide equal confidence for all three different Python versions I guess we'd need to change everything that runs on `35, 36` to `35, 36, 37` and everything that's currently been on `36` only to be `36, 37`. That will totally ruin our CI times though.

Reply:
Yes, seems so.

For ref, this PR (running in my CircleCI, not the `ethereum` org's) [took about an hour](https://circleci.com/workflow-run/897b3219-db10-4102-a59d-cb1ba6900397).

Reply:
We can increase our circle CI capacity so I'd recommend we expand the test suite out to fully cover the python versions we want to test against for all of our runs.

Reply:
I haven't checked the rest of the test suite with Python 3.7.

Do clarify: are you two suggesting it be done in this PR, or a follow-up one?..